testHelpers/styleMock.js

```diff
@@ -0,0 +1 @@
+module.exports = {};
```

Comment:
Put a comment on top of this file explaining why it exists (it mocks out CSS imports for jest)